src/ensembl/src/shared/components/help-popup/HelpPopupBody.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import styles from './HelpPopupBody.scss';
+
+export type HelpVideo = {
+  title: string;
+  description: string;
+  url: string;
+};
+
+type ArticleSummary = {
+  title: string;
+  slug: string;
+  path: string;
+};
+
+export type HelpArticle = {
+  path: string;
```

Comment:
Is path mandatory for all the articles?

Reply:
If you are asking whether the `path` field will always be in the article returned from the api, then yes.

If you are asking whether HelpPopupBody component needs to know about the `path` field, then no.

But currently HelpPopupBody is the place that is describing the shape of the article returned by the api. We may move the type definition to a dedicated types file if you would prefer that.

Reply:
Thanks. Happy to leave the type definition here.